3p/integration.js

```diff
@@ -26,6 +26,7 @@ import './polyfills';
 import {installEmbedStateListener} from './environment';
 import {a9} from '../ads/a9';
 import {adform} from '../ads/adform';
+import {adman} from '../ads/adman';
```

Comment:
Please add an example to `examples/ads.amp.html` and validate that i actually works.